tests/flask_integration/test_member_repository.py

```diff
@@ -68,3 +68,15 @@ def test_get_by_id_returns_none_when_member_does_not_exist(
 ) -> None:
     member = repository.get_by_id(uuid4())
     assert member is None
+
+
+@injection_test
+def test_that_all_members_can_be_retrieved(
```

Comment:
I think we could use an additional test that checks if the number of returned members is equal to the number of created members.